vllm/config.py

```diff
@@ -1319,10 +1324,27 @@ class ParallelConfig:
     worker_cls: str = "auto"
     sd_worker_cls: str = "auto"
 
+    # world_size is TPxPP, it affects the number of workers we create.
     world_size: int = field(init=False)
+    # world_size_across_dp is TPxPPxDP, it is the size of the world
+    # including data parallelism.
+    world_size_across_dp: int = field(init=False)
```

Comment:
Ideally we should use the term `world_size` for `TPxPPxDP`, and `world_size_per_dp` for `TPxPP` to align general impressions. But I guess this would change lots of places...

Reply:
> I guess this would change lots of places...

yes that's so true. this is exactly the reason why i keep the meaning of the original `world_size` and add a new `world_size_across_dp`